contracts/StakingContract.sol

```diff
@@ -70,8 +70,8 @@ contract StakingContract is IStakingContract {
     event AcceptedMigration(address indexed stakeOwner, uint256 amount, uint256 totalStakedAmount);
     event MigratedStake(address indexed stakeOwner, uint256 amount, uint256 totalStakedAmount);
     event MigrationManagerUpdated(address indexed migrationManager);
-    event MigrationDestinationAdded(IStakingContract indexed stakingContract);
-    event MigrationDestinationRemoved(IStakingContract indexed stakingContract);
+    event MigrationDestinationAdded(IMigratableStakingContract indexed stakingContract);
```

Comment:
Shouldn't the events be defined in the interface file?

Reply:
Good idea. I'll move the respective events to the interface file as well.